.github/workflows/release.yml

```
@@ -26,7 +26,7 @@ jobs:
         # an output. It _does_ register that token as a secret so that it will be
         # filtered from log output automatically
         id: generate-token
-        uses: tibdex/github-app-token@586e1a624db6a5a4ac2c53daeeded60c5e3d50fe
+        uses: tibdex/github-app-token@f717b5ecd4534d3c4df4ce9b5c1c2214f0f7cd06
         with:
           app_id: ${{ secrets.APP_ID }}
           private_key: ${{ secrets.APP_PRIVATE_KEY }}
```

If we use this version, we should use the new permissions scoping feature that came with it.

https://github.com/tibdex/github-app-token/blob/f717b5ecd4534d3c4df4ce9b5c1c2214f0f7cd06/action.yml#L15-L16

```suggestion
          private_key: ${{ secrets.APP_PRIVATE_KEY }}
          permissions: >-
            {
              "contents": "write"
            }
```